src/sentry/migrations/0817_update_notificationmessage_constraints_for_action_group_open_period.py

```diff
@@ -74,7 +74,7 @@ class Migration(CheckedMigration):
                 condition=models.Q(
                     ("error_code__isnull", True), ("parent_notification_message__isnull", True)
                 ),
-                name="singular_parent_message_per_rule_fire_history_rule_action_open_period",
+                name="singular_parent_message_per_rfh_ra_open_period",
```

Comment:
It needs to match the existing name, so we can just call it `singular_parent_message_per_rule_fire_history_rule_action_open_`